src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -145,6 +146,7 @@ export const transformTerminalTransaction = (
 
   if (transaction.context?.success?.href) {
     SessionStorage.removeStateHandle();
+    removeUserAuthenticatedCookie();
```

Comment:
This section of code occurs when the user is authenticated, why remove the authenticated cookie here?

Reply:
cleaned up: https://github.com/okta/okta-signin-widget/pull/3434/commits/e0ff2c0d00c8b9d7301fde3c0bb75d9b516ccbd9